zarf.yaml

```diff
@@ -1,7 +1,7 @@
 kind: ZarfPackageConfig
 metadata:
   name: keycloak-identity-config
-  version: "0.0.1"
+  version: dev
```

Comment:
@blancharda just want to double check with you - is this change okay? I think you were using this zarf package for an environment, want to make sure we don't mess with the version if its going to cause pain.

Reply:
We didn't actually end up using this, so no immediate impact here 👍